### PR TITLE
Remove exceptions from phpunit whitelist

### PR DESCRIPTION
**Pull request by jordisala1991**
I am targeting this branch, because this is pedantic.

## Subject

I think coverage is not calculated correctly, lets see

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -12,21 +12,26 @@
          bootstrap="Tests/bootstrap.php"
 >
     <testsuites>
-        <testsuite name="PHPCR Util Test Suite">
-            <directory>./Tests/</directory>
+        <testsuite name="DashboardBundle Test Suite">
```
Comment (OskarStark) [inline phpunit.xml.dist]: wrong name

**Comment by OskarStark** [comment]
@ElectricMaxxx how do we proceed with version eye? can we whitelist the license?

**Comment by dbu** [comment]
regarding version eye i have no idea how to tell it that we are ok with depending on PHP in a PHP project :-( i clicked around in the GUI but did not even find sonata in the projects list.

**Comment by OskarStark** [comment]
Thank you for checking this @dbu 👍

**Pull request merged**